addons/mail/static/src/discuss/core/web/chat_window_patch.xml

```diff
@@ -4,7 +4,7 @@
         <xpath expr="//t[@name='thread content']" position="after">
             <div t-if="!thread" class="d-flex align-items-center m-3">
                 <span class="flex-shrink-0 me-2">To :</span>
-                <ChannelSelector autofocus="props.chatWindow.autofocus" category="store.discuss.chats" onValidate="() => this.close()" multiple="false"/>
+                <DiscussSearch autofocus="props.chatWindow.autofocus" category="store.discuss.chats" onCompleted="() => this.close()" groupChats="false" />
```

Comment:
The limit to `chats` is stupid, I would allow to search for channels too.
Also no reason to not allow to make group chats too.
And even create channels should be fine too...
So really, it can have all the features of sidebar search available!